.gitignore

```diff
@@ -105,6 +105,10 @@ build-install/*
 .cache/*
 .pytest_cache/*
 
+# hypothesis cache #
+####################
+**/.hypothesis
+
```

Comment:
Can/should be removed because the Hypothesis home directory is set by the profile to be a temporary directory?